CONTRIBUTING.md

```diff
@@ -255,6 +255,7 @@ For instance:
 ## PRs & releases
 
 The default branch of the repository is `master`, but each regular (non-hotfix, non-release, non-backport) PR must be merged to `develop`. <br/>
+Because of that all regular PR branches should be derived from `develop` and not `master`. <br/>
 Due to the fact that the default branch is not `develop`, merging of PRs does not close linked issues (you have to close the issues manually). <br/>
 Stacked PRs (Y -> X -> `develop`) must never be merged until their base is finally changed to `develop`.
 They must instead be retargeted to its base's base once their base branch is merged itself (Y -> X -> `develop` => X gets merged => Y -> `develop`).
```

Comment:
Actually! Since ~2019, GitHub retargets PRs automatically in such scenario, pls update while you're here

Reply:
> They must instead be retargeted

This implies that retargeting must be performed **manually**, which is not true since ~2019. I'd rather expect the github blogpost to be mentioned here.